packages/itwin/tree-widget/package.json

```diff
@@ -59,6 +59,8 @@
     "@itwin/core-react": "^3.0.0",
     "@itwin/eslint-plugin": "^3.0.0",
     "@itwin/imodel-components-react": "^3.0.0",
+    "@itwin/itwinui-icons-react": "1.8.0",
```

Comment:
i dont think the icons need to be dev+peer and they can be a direct dep

Reply:
Done